<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddressBookPage</class>
 <widget class="QWidget" name="AddressBookPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Address Book</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>1000</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: QLinearGradient(x1: 0, y1: 0, x2: 1, y2: 0, stop: 0 #121548, stop: 1 #4a0e95);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="winTitle">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>20</y>
          <width>371</width>
          <height>62</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>100</horstretch>
          <verstretch>100</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
background-color: rgba(255, 255, 255,0);</string>
        </property>
        <property name="text">
         <string>    Receive</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelExplanation">
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="toolTip">
      <string>Double-click to edit address or label</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QHeaderView::section {font-size:16px;color:white;height:30px;background-color:#4a0e95}
QHeaderView::section:checked
{
    font-size:16px;color:white;height:30px;background-color:#4a0e95
}</string>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>60</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <item>
      <widget class="QPushButton" name="newAddress">
       <property name="toolTip">
        <string>Create a new address</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(15, 15, 54);
color: white;
border-radius:10px;
height:30px;
width:120px;</string>
       </property>
       <property name="text">
        <string>&amp;New Address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyAddress">
       <property name="toolTip">
        <string>Copy the currently selected address to the system clipboard</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(15, 15, 54);
color: white;
border-radius:10px;
height:30px;
width:120px;</string>
       </property>
       <property name="text">
        <string>&amp;Copy Address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showQRCode">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(15, 15, 54);
color: white;
border-radius:10px;
height:30px;
width:120px;</string>
       </property>
       <property name="text">
        <string>Show &amp;QR Code</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="signMessage">
       <property name="toolTip">
        <string>Sign a message to prove you own a zoin address</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(57, 0, 130);
color: white;
border-radius:10px;
height:30px;
width:120px;</string>
       </property>
       <property name="text">
        <string>Sign &amp;Message</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="verifyMessage">
       <property name="toolTip">
        <string>Verify a message to ensure it was signed with a specified zoin address</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(57, 0, 130);
color: white;
border-radius:10px;
height:30px;
width:120px;</string>
       </property>
       <property name="text">
        <string>&amp;Verify Message</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteAddress">
       <property name="toolTip">
        <string>Delete the currently selected address from the list</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(57, 0, 130);
color: white;
border-radius:10px;
height:30px;
width:120px;</string>
       </property>
       <property name="text">
        <string>&amp;Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zerocoinMintButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(57, 0, 130);
color: white;
border-radius:10px;
height:30px;
width:120px;</string>
       </property>
       <property name="text">
        <string>Mint Zerocoin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zerocoinSpendButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(57, 0, 130);
color: white;
border-radius:10px;
height:30px;
width:120px;</string>
       </property>
       <property name="text">
        <string>Spend Zerocoin</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="toolTip">
        <string>Export the data in the current tab to a file</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(57, 0, 130);
color: white;
border-radius:10px;
height:30px;
width:120px;</string>
       </property>
       <property name="text">
        <string>&amp;Export</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
